build/build.fsproj

```diff
@@ -0,0 +1,38 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net9.0</TargetFramework>
+    <WarnOn>3390;$(WarnOn)</WarnOn>
+    <IsPackable>false</IsPackable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Content Include="Properties\launchSettings.json" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <Compile Include="Changelog.fs" />
+    <Compile Include="Fornax.fs" />
+    <Compile Include="build.fs" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Argu" />
+    <PackageReference Include="Fake.Api.GitHub" />
+    <PackageReference Include="Fake.BuildServer.GitHubActions" />
+    <PackageReference Include="Fake.Core.Environment" />
+    <PackageReference Include="Fake.Core.Process" />
+    <PackageReference Include="Fake.Core.ReleaseNotes" />
+    <PackageReference Include="Fake.Core.Target" />
+    <PackageReference Include="Fake.DotNet.AssemblyInfoFile" />
+    <PackageReference Include="Fake.DotNet.Cli" />
+    <PackageReference Include="Fake.DotNet.Paket" />
+    <PackageReference Include="Fake.IO.FileSystem" />
+    <PackageReference Include="Fake.Tools.Git" />
+    <PackageReference Include="FSharp.Control.Reactive" />
+    <PackageReference Include="MSBuild.StructuredLogger" />
+    <PackageReference Include="NuGet.Common" />
+    <PackageReference Include="NuGet.Protocol" />
+    <PackageReference Include="Octokit" />
```

Comment:
@xperiandri what is Octokit dep for?

Reply:
Think the question was why does this project directly reference the Octokit package if it's used via the FAKE module rather than being called directly, and then I noticed that the referenced versions are different and wondered if it was done to intentionally take the latest version?

(And then I noticed that the v14 release notes list some breaking changes, and recalled that has caused issues with the FAKE module in the past - I don't know if those changes will effect what it's being used for here though)

Reply:
@Numpsy, I agree that we need to update it in Fake first. So I removed it here